README.md

```diff
@@ -16,202 +21,54 @@ This is an early version of the plugin, my primary goal was to overcome the bann
-AdmobBanner(
-  adUnitId: getBannerAdUnitId(),
-  adSize: AdmobBannerSize.BANNER,
-  listener: (AdmobAdEvent event, Map<String, dynamic> args) {
-    switch (event) {
-      case AdmobAdEvent.loaded:
-        print('Admob banner loaded!');
-        break;
-
-      case AdmobAdEvent.opened:
-        print('Admob banner opened!');
-        break;
-
-      case AdmobAdEvent.closed:
-        print('Admob banner closed!');
-        break;
-
-      case AdmobAdEvent.failedToLoad:
-        print('Admob banner failed to load. Error code: ${args['errorCode']}');
-        break;
-    }
-  }
-)
-```
-
-### Showing Interstitial Ads
-
-```dart
-// First, create an interstitial ad
-AdmobInterstitial interstitialAd = AdmobInterstitial(
-  adUnitId: getInterstitialAdUnitId(),
-);
-
-// Interstitials must be loaded before shown.
-// Make sure to always load them ahead so they can be
-// available to be shown immediately to the user
-interstitialAd.load();
-
-// Check if the ad is loaded and then show it
-if (await interstitialAd.isLoaded) {
-  interstitialAd.show();
-}
-
-// Finally, make sure you dispose the ad if you're done with it
-interstitialAd.dispose();
-```
-
-### Listening to Interstitial Ads events
-
-Same as banner ads, you can attach a listener to interstitials too. For example if you wish to:
-- Show the interstitial as soon as it is loaded
-- Load a new ad and show it as soon as the old one is closed
-- Get your users stuck in an interstitial ads loop to maximize revenue
-- Get your Admob account banned and lose all your earnings
-
-You can do something like this (Don't do it!):
+3. Import it
+Now in your Dart code, you can use:
 
 ```dart
-AdmobInterstitial interstitialAd;
-
-interstitialAd = AdmobInterstitial(
-  adUnitId: getInterstitialAdUnitId(),
-  listener: (AdmobAdEvent event, Map<String, dynamic> args) {
-    if (event == AdmobAdEvent.loaded) interstitialAd.show();
-    if (event == AdmobAdEvent.closed) interstitialAd.load();
-    if (event == AdmobAdEvent.failedToLoad) {
-      // Start hoping they didn't just ban your account :)
-      print("Error code: ${args['errorCode']}");
-    }
-  },
-);
+import 'package:admob_flutter/admob_flutter.dart';
 ```
+  
 
-### Showing Reward Ads
-
-```dart
-// First, create a reward ad
-AdmobReward rewardAd = AdmobReward(
-  adUnitId: getRewardBasedVideoAdUnitId(),
-);
-
-// Reward ads must be loaded before shown.
-// Make sure to always load them ahead so they can be
-// available to be shown immediately to the user
-rewardAd.load();
-
-// Check if the ad is loaded and then show it
-if (await rewardAd.isLoaded) {
-  rewardAd.show();
-}
-
-// Finally, make sure you dispose the ad if you're done with it
-rewardAd.dispose();
-```
+### Supported Platforms
+- `0.3.0` >= iOS
+- `0.2.0` >= AndroidX
+- FlutterSdk >=2.1.0 < 3.0.0
 
-### Listening to Reward Ads events
+### Supported Admob features
+- Banner Ads
+- Interstitial Ads
+- Reward Ads
+- Native Ads (Coming soon)
+- Using targeting info (Coming soon)
```

Comment:
Per https://github.com/YoussefKababe/admob_flutter/issues/7 I don't believe targeting will be implemented.

Reply:
I didn't see that, just removed it. @kmcgill88